numpy/lib/tests/test_stride_tricks.py

```diff
@@ -57,6 +58,19 @@ def test_same():
     assert_array_equal(x, bx)
     assert_array_equal(y, by)
 
+def test_broadcast_kwargs():
+    # ensure that a TypeError is appropriately raised when
+    # np.broadcast_arrays() is called with any keyword
+    # argument other than 'subok'
+    x = np.arange(10)
+    y = np.arange(10)
+
+    with pytest.raises(TypeError) as excinfo:
+        broadcast_arrays(x, y, dtype='float64')
+
+    assert ('broadcast_arrays() got an unexpected keyword' in
+            str(excinfo.value))
```

Comment:
Hmm, I think we are still using `assert_`. We should probably have a agreement about how to proceed with the new options pytest gives us.

Reply:
For reference, SciPy [had this discussion](https://github.com/scipy/scipy/issues/8583) a few months ago--not sure if we should similarly open an issue or ask on the mailing list? They decided in favor of allowing pytest `assert` usage in new PRs, but didn't mandate a mass migration.

Perhaps I'll just revert here for now?

Reply:
If we were using pytest more broadly here, I think you could spell this `excinfo.match(r'broadcast_arrays\(\) got an unexpected keyword')`